cmd/kubeadm/app/images/images_test.go

```diff
@@ -27,8 +27,8 @@ import (
 )
 
 const (
-	testversion = "v10.1.2-alpha.1.100+0123456789abcdef+SOMETHING"
-	expected    = "v10.1.2-alpha.1.100_0123456789abcdef_SOMETHING"
+	testversion = "v10.1.2-alpha.1.100+0123456789abcdef"
+	expected    = "v10.1.2-alpha.1.100_0123456789abcdef"
```

Comment:
the semver parser that `constants.EtcdSupportedVersion(` uses (from api-machinery) actually does not support this, despite it being a valid semver e.g. according to https://jubianchi.github.io/semver-check/#/version/v10.1.2-alpha.1.100%2B0123456789abcdef%2BSOMETHING